sdk/ml/azure-ai-ml/azure/ai/ml/constants/_common.py

```diff
@@ -182,6 +182,9 @@
     "Spark job will only install the packages defined in the Conda configuration. It "
     "will not create a docker container using the image defined in the environment."
 )
+API_VERSION_KEY = "ApiVersion"
```

Comment:
The names for those constants are too generic. Perhaps consider adding a "CONNECTION_" prefix for clarity.